examples/simple/asyncapi.yaml

```diff
@@ -0,0 +1,39 @@
+asyncapi: 2.4.0
+info:
+  title: Test Payload
+  version: 1.0.0
+  description: Test payload schema
+servers:
+  http:
+    url: http://localhost:8080
+    protocol: http
+channels:
+  test:
+    bindings:
+      http:
+        bindingVersion: 0.1.0
+        type: request
+        method: GET
+        query:
+          type: object
+          properties:
+            name:
+              type: string
```

Comment:
HTTP should support multiple methods for which I did start an initial work https://github.com/asyncapi/bindings/pull/131 I would suggest you look into it and move it forward.